packages/pn-pa-webapp/src/components/Notifications/FilterNotificationsFormBody.tsx

```diff
@@ -87,6 +154,14 @@ const FilterNotificationsFormBody = ({
     );
   };
 
+  const mb = isMobile ? '20px' : 0;
+
+  const startErr = formikInstance.errors?.startDate as string | undefined;
+  const endErr = formikInstance.errors?.endDate as string | undefined;
+
+  // verify it's a "max range exceeded" error, otherwise no helperText is shown
+  const helperText = startErr && endErr && startErr === endErr ? startErr : undefined;
```

Comment:
I don't understand why we cant use formikInstance.errors?.startDate and formikInstance.errors?.endDate directly in the jsx